src/App.tsx

```diff
@@ -1,167 +1,43 @@
-import { Loader } from './components/Loader';
-
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
+import { Navigate, NavLink, Outlet, useLocation } from 'react-router-dom';
+import classNames from 'classnames';
+
+const getActiveLink = ({ isActive }: { isActive: boolean }) =>
+  classNames('navbar-item', { 'has-background-grey-lighter': isActive });
+
+export const App = () => {
+  const { pathname } = useLocation();
+
+  if (pathname === '/home') {
```

Comment:
You should do it in Root, using `Navigate` component as element (google React Router documentation about this component)